v4/source/users.yaml

```diff
@@ -2133,6 +2792,29 @@
               MfaCode:        mfaCode,
               LdapLoginId:    ldapLoginID,
             })
+        - lang: PHP
+          source: |
+            require 'vendor/autoload.php';
+
+            use \Gnello\Mattermost\Driver;
+
+            $container = new \Pimple\Container([
+                "driver" => [
+                    "url" => "https://your-mattermost-url.com",
+                    "login_id" => "email@domain.com",
+                    "password" => "Password1",
+                  ]
+            ]);
+
+            $driver = new Driver($container);
+            $driver->authenticate();
+
+            $result = $driver->getUserModel()->switchLoginMethod([
+                "current_service" => "email",
+                "new_service" => "gitlab",
+                "email" => "test@domain.com",
+                //...
```

Comment:
Unless there is a strong reason for leaving the rest of the values out of the example, I would put all the values on the Go example.